### DISCUSSION TITLE
Is multimaterial supported?

### DISCUSSION THREAD

**Discussion by Contrasens** [Donkie/Spoolman#232] [General]
Thinking to upgrade my printer to use [ROME](https://github.com/HelgeKeck/rome) which is a multimeterial extrusion system for RatRig (but not only) 3d printers. I find Spoolman very cool, just asking myself if is supports somehow multimaterial systems.

**Comment by Donkie** [comment]
Spoolman supports it fine, but the printer probably doesn't. I'm not sure about the ecosystem for that printer so I can't tell.

**Reply by Contrasens** [reply]
I guess with multimaterial systems one has to first declare which spools are in which slots, then at every material change (Klipper macro call?) Spoolman has to be informed to change the active spool....right?

**Reply by Donkie** [reply]
Yup, exactly
